### PR TITLE
Working vendors

### PR DESCRIPTION
**Pull request by DmitriyRubetskoy**
Changed the chocolate dispenser machine so it actually dispenses sweets

### CODE REVIEW

**Comment by SweptWasTaken** [comment]
Your title should be more descriptive than this. There's a lot of vending machines in the game, and "working" could mean anything.

File: Resources/Prototypes/Catalog/VendingMachines/snack.yml [hunk 1]
```diff
@@ -1,5 +1,13 @@
 - type: vendingMachineInventory
   id: Getmore Chocolate Corp
   name: Getmore Chocolate Corp
-  description: A vending machine containing snacks.
```
Comment (SweptWasTaken) [inline Resources/Prototypes/Catalog/VendingMachines/snack.yml:4]: Why did you remove the description?

**Review by SweptWasTaken — CHANGES_REQUESTED** [review]

**Review by PJB3005 — CHANGES_REQUESTED** [review]
Your IDE absolutely butchered all the project files. Please clear all that stuff out of your PR.

**Comment by DmitriyRubetskoy** [comment]
I will make another pull request with proper name and files

**Pull request closed without merging**